src/views/Home.vue

```diff
@@ -1,20 +1,43 @@
 <template>
   <div class="home">
-    <router-link to="/about">
-      <img class="rotating" alt="Mesh logo" src="../assets/mesh-logo.png">
-    </router-link>
-    <HelloWorld msg="Welcome to Mesh"/>
+    <h1>This is a 🏡 component.</h1>
+    <div class="main">
+      <section class="left-section">
+        <h1 class="module">Module 1</h1>
+        <h1 class="module">Module 2</h1>
+        <h1 class="module">Module 3</h1>
+        <h1 class="module">Module 4</h1>
+      </section>
+      <section class="right-section">
+        <section class="projects-container">
+          <h1 class="project">Project 1</h1>
+          <h1 class="project">Project 2</h1>
+          <h1 class="project">Project 3</h1>
+          <h1 class="project">Project 4</h1>
+        </section>
+
+        <section class="feedback-container">
+          <h2>Feedback goes here</h2>
```

Comment:
Right on. So I think I just need to insert an instance of my `<StudentFeedback>` component here instead of the `section`.

Reply:
Yep!  We just sub in your component for my section.  🎉 🎉 🎉